src/java/org/apache/cassandra/io/sstable/SSTable.java

```diff
@@ -155,9 +155,9 @@ public DecoratedKey decorateKey(ByteBuffer key)
      */
     public static DecoratedKey getMinimalKey(DecoratedKey key)
```

Comment:
Nit: could this be renamed? E.g. to `onHeapMinimized`? Or something in the vein of `retainable` like the clusterings?

Reply:
actually, no need this method to be in `SSTable`, I'll move it to `DecoratedKey`

Reply:
looks good